sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -271,7 +274,7 @@ List the certificates in the key vault by calling `listPropertiesOfCertificates`
 ```Java
 // The List Certificates operation returns certificates without their full properties, so for each certificate returned we call `getCertificate` to get all its attributes excluding the policy.
 certificateAsyncClient.listPropertiesOfCertificates()
-    .subscribe(certificateProperties -> certificateAsyncClient.getCertificate(certificateProperties.getName(),
+    .subscribe(certificateProperties -> certificateAsyncClient.getCertificateVersion(certificateProperties.getName(),
```

Comment:
[Line](https://github.com/Azure/azure-sdk-for-java/pull/10118/files#diff-824e08966bb186e6f79351c1a042d490R272) update to `Key Vault`.